code/game/objects/machinery/vending/loadout_vendor.dm

```diff
@@ -1,5 +1,5 @@
 /obj/machinery/loadout_vendor
-	name = "automated vendor"
+	name = "\improper Automated Loadout Vendor"
```

Comment:
Huh? but this isnt a proper name it should be lowercase

Reply:
Yeah, but weapons vendor follow the same format despite not being proper nouns.
Should I change that instead?

Reply:
Yea

Reply:
Done